content/blog/other/potw.md

```diff
@@ -3,6 +3,8 @@ title: Playlist of the week
 author: Mathias Bøe
 tags: playlist, week
 date: 2020-09-30
+color: "#1DB954"
```

Comment:
lmao, `#BADA55` plz

Reply:
Gotta use the official spotify green for potw man